skbio/diversity/alpha/_base.py

```diff
@@ -847,6 +848,51 @@ def osd(counts):
     counts = _validate_counts_vector(counts)
     return observed_otus(counts), singles(counts), doubles(counts)
 
+@experimental(as_of="0.4.0")
+def pielou(counts):
+    """Calculate Pielou's Evenness (Shannon index / ln(number of OTUs).
```

Comment:
Can you remove the equation from this summary line? numpydoc recommends putting this info in the body of the docstring (like you already have). A summary line such as the following should suffice:

```
Calculate Pielou's evenness index J'.
```